app/importers/envirotech/background_link_data.rb

```diff
@@ -21,7 +21,7 @@ def initialize(resource = ENDPOINT)
     end
 
     def import
-      data = fetch_data
+      data = @resource =~ URI.regexp ? fetch_data : JSON.parse(File.open(@resource).read)
```

Comment:
What I don't like about this approach is that this is essentially behavior that only happens during tests, while the fetch_data is only run in production.

I'd rather the "from_web" returned a resource file which could be passed in so the behavior would be the same.
2 steps, but the same. It would also be isolated and testable.